.github/actions/help-command/action.yml

```
@@ -80,12 +83,14 @@ runs:
             '3. Open an issue in this repository',
           ].join('\n');
 
-          const context = github.context;
-          if (context.eventName === 'issue_comment') {
+          const issueNumber = inputs['issue-number'] || 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)